### PR TITLE
I18n

### PR DESCRIPTION
**Pull request by kosteman**

### CODE REVIEW

**Review by clicman — APPROVED** [review]

**Review by voronsp — APPROVED** [review]

File: src/main/resources/i18n/ru/sbtqa/tag/apifactory/stepdefs/GenericStepDefs/en.properties [hunk 1]
```diff
@@ -0,0 +1,4 @@
+userSendRequestNoParams=^user sends request (?:for|to|about) \\((.*?)\\)$
+userSendRequestTableParam=^user sends request (?:for|to|about) \\((.*?)\\) with parameters$
```
Comment (0I0u3uk) [inline src/main/resources/i18n/ru/sbtqa/tag/apifactory/stepdefs/GenericStepDefs/en.properties]: Maybe:
```
^user sends request (?:for|to|about) \\((.*?)\\) with parameters.*$
```

File: src/main/resources/i18n/ru/sbtqa/tag/apifactory/stepdefs/GenericStepDefs/en.properties [hunk 2]
```diff
@@ -0,0 +1,4 @@
...
+userValidate=^system returns \"([^\"]*)\"$
+userValidateTable=^system returns \"([^\"]*)\" with parameters$
```
Comment (0I0u3uk) [inline src/main/resources/i18n/ru/sbtqa/tag/apifactory/stepdefs/GenericStepDefs/en.properties]: And here

File: src/main/resources/i18n/ru/sbtqa/tag/apifactory/stepdefs/GenericStepDefs/ru.properties [hunk 3]
```diff
@@ -0,0 +1,5 @@
+userSendRequestNoParams=^(?:пользователь |он |) отправляет запрос \\((.*?)\\)$
+userSendRequestTableParam=^(?:пользователь |он |) отправляет запрос \\((.*?)\\) с параметрами$
```
Comment (0I0u3uk) [inline src/main/resources/i18n/ru/sbtqa/tag/apifactory/stepdefs/GenericStepDefs/ru.properties]: .*

File: src/main/resources/i18n/ru/sbtqa/tag/apifactory/stepdefs/GenericStepDefs/ru.properties [hunk 4]
```diff
@@ -0,0 +1,5 @@
...
+userValidate=^система возвращает ответ \"([^\"]*)\"$
+userValidateTable=^система возвращает ответ \"([^\"]*)\" с параметрами$
```
Comment (0I0u3uk) [inline src/main/resources/i18n/ru/sbtqa/tag/apifactory/stepdefs/GenericStepDefs/ru.properties]: .*

File: src/main/resources/i18n/ru/sbtqa/tag/apifactory/stepdefs/GenericStepDefs/en.properties (hunk 1 above)
Comment (kosteman) [reply src/main/resources/i18n/ru/sbtqa/tag/apifactory/stepdefs/GenericStepDefs/en.properties]: For what?

`^user sends request (?:for|to|about) \\((.*?)\\) with parameters$ `
catched steps with cucumber DataTable like
```
* user sends request to (add new popular brand) with parameters
      | brandId | 777   |
      | name    | Lotus |
```
Whereas, 
`^user sends request (?:for|to|about) \\((.*?)\\) with parameters.*$`
will catch everything like
`user sends request to (add new popular brand) with parameters !@$SWFPCV<A{}}.][321]"`

Comment (0I0u3uk) [reply src/main/resources/i18n/ru/sbtqa/tag/apifactory/stepdefs/GenericStepDefs/en.properties]: Our users sometime write "user sends request to (add new popular brand) with parameters:" 
And dont understand why it doesnt work. That's why I suggested it

**Review by 0I0u3uk — APPROVED** [review]

**Pull request merged**